.github/workflows/build.yml

```diff
@@ -16,7 +16,7 @@ permissions:
   contents: write
 
 env:
-  version: m138-c1caa59365c-1
+  version: m138-c1caa59365c-2
```

Comment:
There is a regression in this commit. Let's revert to `9e6b5bff162` for now to avoid blocking skiko releases